cluster/olm/bundle/manifests/upbound-bootstrapper.clusterrole.yaml

```diff
@@ -1,28 +0,0 @@
-# Generated by https://github.com/upbound/olm-bundle
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  labels:
-    app.kubernetes.io/component: bootstrapper
-    app.kubernetes.io/instance: universal-crossplane
-    app.kubernetes.io/managed-by: Helm
-    app.kubernetes.io/name: crossplane
-    app.kubernetes.io/version: 0.0.1
-    helm.sh/chart: universal-crossplane-0.0.1
-  name: upbound-bootstrapper
-rules:
-- apiGroups:
-  - ""
-  resourceNames:
-  - kube-system
-  resources:
-  - namespaces
-  verbs:
-  - get
-- apiGroups:
-  - ""
-  resources:
-  - namespaces
-  verbs:
-  - list
-  - watch
```

Comment:
all these resources are removed from the olm folder because the bootstrap is disabled by default, is this ok? 🤔

Reply:
Yes. Currently we only need bootstrapper for installations through AWS marketplace. OLM bundle is not the artifact used for that installation method.